### PR TITLE
Page template updates

### PR DESCRIPTION
**Pull request by wesrice**
- Adds `Alert` component
- Updates h2 and h3 styles to have more visual distinction

### CODE REVIEW

File: src/components/Alert.a11y.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    );
+    const results = await pa11y(component, {
+      ignore: [
+        'region',
+      ],
+    });
+    expect(results).toHaveNoPa11yViolations();
+  });
+
+  xit('type of `info` has no Pa11y violations', async () => {
```
Comment (kevcenteno) [inline src/components/Alert.a11y.tsx:23]: Why are these tests disabled?

File: src/components/Mdx/Pre/Code.dark-theme.css [hunk 2]
```diff
@@ -0,0 +1,274 @@
+/* stylelint-disable */
```
Comment (kevcenteno) [inline src/components/Mdx/Pre/Code.dark-theme.css:1]: Why did you move the styles out of the sass file and into a css file?  Does `yarn develop` still cactch changes to css files?

**Review by kevcenteno — CHANGES_REQUESTED** [review]

File: src/components/Mdx/Pre/Code.dark-theme.css (hunk 2 above)
Comment (wesrice) [reply src/components/Mdx/Pre/Code.dark-theme.css:1]: This is actually an incompatibility issue with https://github.com/wesrice/jest-style-transformers. For some reason, this Scss file fails to compile when transformed by `jest-style-transformers`.  I haven't been able to get to the root cause yet, so I just switched it from scss to css to circumvent the issue, especially since this isn't a scss module and the styles are at a global scope anyways.

File: src/components/Alert.a11y.tsx (hunk 1 above)
Comment (wesrice) [reply src/components/Alert.a11y.tsx]: I disabled these when troubleshooting the css issue below, and forgot to reenable them. I pushed a fix to enable them.
